common/tier4_vehicle_rviz_plugin/src/tools/acceleration_meter.cpp

```diff
@@ -0,0 +1,260 @@
+    std::max(std::fabs(acceleration_x) - meter_min_acceleration_, 0.0) /
+      (meter_max_acceleration_ - meter_min_acceleration_),
+    1.0);
+  const float theta =
+    (acceleration_ratio * (meter_max_angle_ - meter_min_angle_)) + meter_min_angle_ + M_PI_2;
+
+  painter.setPen(QPen(white_color, hand_width_, Qt::SolidLine));
+  painter.drawLine(
+    w * 0.5, h * 0.5,
+    (w * 0.5) +
+      (static_cast<float>(w) * 0.5 - (static_cast<float>(hand_width_) * 0.5)) * std::cos(theta),
+    (h * 0.5) +
+      (static_cast<float>(h) * 0.5 - (static_cast<float>(hand_width_) * 0.5)) * std::sin(theta));
+
+  painter.setPen(QPen(white_color, line_width_, Qt::SolidLine));
+  painter.drawLine(min_range_line_.x0, min_range_line_.y0, min_range_line_.x1, min_range_line_.y1);
+  painter.drawLine(max_range_line_.x0, max_range_line_.y0, max_range_line_.x1, max_range_line_.y1);
+  painter.drawArc(
+    outer_arc_.x0, outer_arc_.y0, outer_arc_.x1, outer_arc_.y1, outer_arc_.start_angle * 16,
+    outer_arc_.end_angle * 16);
+  painter.drawArc(
+    inner_arc_.x0, inner_arc_.y0, inner_arc_.x1, inner_arc_.y1, inner_arc_.start_angle * 16,
+    inner_arc_.end_angle * 16);
+
+  // text
+  QColor text_color;
+  if (std::abs(acceleration_x) > property_emergency_threshold_->getFloat()) {  // Write in Red if
+                                                                               // acceleration is
+                                                                               // over emergency
+                                                                               // threshold.
+    text_color = property_emergency_text_color_->getColor();
+  } else {
+    text_color = property_normal_text_color_->getColor();
+  }
+  text_color.setAlpha(255);
+  painter.setPen(QPen(text_color, static_cast<int>(2), Qt::SolidLine));
+  QFont font = painter.font();
+  font.setPixelSize(
+    std::max(static_cast<int>(static_cast<double>(w) * property_value_scale_->getFloat()), 1));
+  font.setBold(true);
+  painter.setFont(font);
+  std::ostringstream acceleration_ss;
+  // Write acceleration in m/s^2 for debugging usage.
+  acceleration_ss << std::fixed << std::setprecision(2) << acceleration_x << "m/s^2";
+  painter.drawText(
+    0, std::min(property_value_height_offset_->getInt(), h - 1), w,
+    std::max(h - property_value_height_offset_->getInt(), 1), Qt::AlignCenter | Qt::AlignVCenter,
+    acceleration_ss.str().c_str());
+
+  // Write Labels
+  QColor label_color(property_normal_text_color_->getColor());
+  text_color.setAlpha(255);
+  painter.setPen(QPen(label_color, static_cast<int>(2), Qt::SolidLine));
+  QFont label_font = painter.font();
+  font.setPixelSize(
+    std::max(static_cast<int>(static_cast<double>(w) * property_label_scale_->getFloat()), 1));
+  font.setBold(true);
+  painter.setFont(font);  // Use the font as the meter panel but smaller.
+  std::ostringstream label_ss;
+  label_ss << std::fixed << "Acceleration Meter";
```

Comment:
IMO, "Acceleration Meter" is not needed for simplicity like other meters

Reply:
Agree. And sorry I should have checked more carefully for the meter hand computation. These two issues were refined in new pushes.

Reply:
Thank you ! 
(It is ok to contribute tomorrow as a partime job. :smile:  )